modules/api/src/main/java/org/apache/ignite/schema/definition/ColumnType.java

```diff
@@ -203,14 +203,14 @@ public static TemporalColumnType time(int precision) {
     }
 
     /**
-     * Returns timezone-free datetime encoded as (date, time) with the default time precision of 6 (microseconds).
+     * Returns timezone-free datetime encoded as (date, time) with the default time precision of 0 (seconds).
      *
      * @return Native type.
-     * @see TemporalColumnType#DEFAULT_PRECISION
+     * @see TemporalColumnType#DEFAULT_TIME_PRECISION
      * @see #datetime(int)
      */
     public static TemporalColumnType datetime() {
-        return new TemporalColumnType(ColumnTypeSpec.DATETIME, TemporalColumnType.DEFAULT_PRECISION);
+        return new TemporalColumnType(ColumnTypeSpec.DATETIME, TemporalColumnType.DEFAULT_TIME_PRECISION);
```

Comment:
I believe this here should be TIMESTAMP_PRECISION, since DATETIME represented by a SQL TIMESTAMP type